### PR TITLE
Add additional test for volcano_plot component through its demo app

### PR DESCRIPTION
**Pull request by Bachibouzouk**
## Description of changes 

Add more tests for the volcano plot and fulfill #103 

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [ ] I have completed all of the [steps to take before merging](before_merging.md)
- [ ] I know how to [deploy to DDS](deploying_to_dds.md)

### CODE REVIEW

File: tests/test_volcano_plot.py [hunk 1]
```diff
@@ -1,16 +1,74 @@
...
 from .test_common_features import access_demo_app
 
 APP_NAME = os.path.basename(__file__).replace('test_', '').replace('.py', '')
 
 
+LAYOUT = html.Div(
+    id='-test-graph-div',
+    children=[
+        dcc.Graph(
+            id='test-graph',
```
[the remark this answers is not in the document]
Comment (Bachibouzouk) [reply tests/test_volcano_plot.py]: Normally not as each test is built and teared down in a given thread and is not used simultaneously, but I can change it to make sure :)

**Comment by shammamah** [comment]
@Bachibouzouk CircleCI is showing a failure for the python test -- should be addressed before merge

File: tests/test_volcano_plot.py [hunk 2]
```diff
@@ -1,16 +1,74 @@
...
+        :param nclicks: (string) html.Button 'n_clicks' Input
+        :param param_name: (string) dcc.Input 'value' State
+        :param param_value: (string) dcc.Input 'value' State
+        :param param_type: (string) one of PARAM_TYPES keys
+            default: None
+        :param dataset: (panda DataFrame): a DataFrame with volcano plot data
+        :return: a dash_bio.VolcanoPlot instance (which is a plotly.graph_objs.Figure instance)
+    """
+    answer = {'data': [], 'layout': {}}
+    # avoid triggering the
```
[the remark this answers is not in the document]
Comment (Bachibouzouk) [reply tests/test_volcano_plot.py]: Fixed in 2513aa6f85e79b38e83fac1691feac7b96d89d01 :)

File: tests/test_volcano_plot.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bachibouzouk) [reply tests/test_volcano_plot.py]: Fixed in ed85554d2a7a824fa678c82268ab19ff228fbb26 :)

**Comment by Bachibouzouk** [comment]
> @Bachibouzouk CircleCI is showing a failure for the python test -- should be addressed before merge

@shammamah It was a server error, I rerun the tests and it passed :)

**Comment by shammamah** [comment]
💃

**Pull request merged**